packages/app/cypress/e2e/index.cy.ts

```diff
@@ -219,6 +219,111 @@ describe('App: Index', () => {
       })
     })
 
+    context('ts project with default spec pattern', () => {
+      beforeEach(() => {
+        cy.scaffoldProject('no-specs-no-storybook')
+        cy.openProject('no-specs-no-storybook')
+
+        cy.withCtx(async (ctx) => {
+          await ctx.actions.file.writeFileInProject('tsconfig.json', '{}')
+        })
+
+        cy.openProject('no-specs-no-storybook')
+
+        cy.startAppServer('e2e')
+        cy.__incorrectlyVisitAppWithIntercept()
```

Comment:
Unrelated to this PR but the name of this method is confusing! We should either name, document, or both.